platform/ui/src/gatsby-theme-docz/tailwind.css

```diff
@@ -22,3 +22,14 @@
 .ohif-scrollbar::-webkit-scrollbar-thumb:window-inactive {
   @apply bg-primary-main;
 }
+
+/* TOOLTIP WORKAROUND FOR ARROW UP */
+.tooltip::before {
```

Comment:
Is this going to be used on all tooltips? maybe worth adding a custom class for this scenario.

Reply:
Yes! I was only thinking if we have in the future tooltip position like `up`, `left`, `bottom`...
But for now, we have only this kind of Tooltip.